jsonschema-core.xml

```diff
@@ -791,20 +791,22 @@
 
             <section title='Schema References With "$ref"'>
                 <t>
-                    The "$ref" keyword is used to reference a schema, and provides the ability to
-                    validate recursive structures through self-reference.
+                    The "$ref" keyword can be used to reference a schema which is to be applied to the
+                    current instance location. "$ref" is an assertion key word, which MUST produce a boolean
+                    assertion result when the resolved schema is applied.
                 </t>
+                <cref anchor="REF1" source="BH">
+                    The use of "$ref" MUST NOT effect adjacent keywords.
+                    Given previously the use of "$ref" would negate the use of other keywords in that object,
```

Comment:
I think that most of this can be made clear by thoroughly describing the applicator behavior.  Otherwise I would put this in the appendix for people who want to understand the changes, as noted in another comment.

Reply:
I guess we should discuss this as it looks to me like your opinion on applicator or assertation for $ref has changed since the issue was raised.

Reply:
I'm not aware of my opinion changing although I certainly accept that I may have been unclear.